planners/astar/planner.c

```diff
@@ -93,14 +94,19 @@ static int trajectory_get_route_update(ctrl_t* ctrl, const pose_t* robot_pose,
             }
             /* Update current path targeted position in case it has changed */
             current_path_pos = path_get_current_path_pos(path);
+
             /* As current path pose could have changed, avoidance graph needs
-             * to be recomputed */
+                * to be recomputed */
```

Comment:
not aligned

Reply:
The diff is weird, but the code is aligned...